eo-maven-plugin/src/main/java/org/eolang/maven/SafeMojo.java

```diff
@@ -129,6 +129,27 @@ abstract class SafeMojo extends AbstractMojo {
     @Parameter(property = "eo.placedFormat", required = true, defaultValue = "csv")
     protected String placedFormat = "csv";
 
+    /**
+     * The path to a text file where paths of generated java files per EO program.
+     * @checkstyle MemberNameCheck (7 lines)
+     * @checkstyle VisibilityModifierCheck (10 lines)
+     * @since 0.11.0
+     */
+    @Parameter(
+        property = "eo.transpiled",
+        required = true,
+        defaultValue = "${project.build.directory}/eo/transpiled.csv"
+    )
+    protected File transpiled;
+
+    /**
+     * Format of "transpiled" file ("json" or "csv").
+     * @checkstyle MemberNameCheck (7 lines)
+     * @checkstyle VisibilityModifierCheck (5 lines)
+     */
+    @Parameter(property = "eo.placedFormat", required = true, defaultValue = "csv")
```

Comment:
@Karina5005 should be `transpiledFormat`

Reply:
fixed